paper.md

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Introduction to deep learning: Carpentries-style hands-on lesson material for introducing researchers to deep learning'
+tags:
+  - Python
+  - deep learning
+  - machine learning
+  - Keras
+  - neural networks
+authors:
+  - name: Sven A. van der Burg
+    orcid: 0000-0003-1250-6968
+    affiliation: 1 # (Multiple affiliations must be quoted, like "1, 2")
+  - name: Anne Fouilloux
+    orcid: 0000-0002-1784-2920
+    affiliation: 2
+  - name: Florian Huber
+    orcid: 0000-0002-3535-9406
+    affiliation: "1, 3"
+  - name: Dafne van Kuppevelt
+    affiliation: 1
```

Comment:
```suggestion
    orcidid: 0000-0002-2662-1994
    affiliation: 1
```

Reply:
Thanks for adding! FYI it should be orcid and not orcidid (then the ID is double). But it is confusing :)